anomalib/models/patchcore/config.yaml

```diff
@@ -14,7 +14,7 @@ dataset:
     use_random_tiling: False
     random_tile_count: 16
   image_size: 224
-  train_batch_size: 32
+  train_batch_size: 1
```

Comment:
Is this by choice or an artefact of debugging?

Reply:
Good catch. It was an artifact. Fixed it.